packages/components/src/organisms/BannerText/BannerTextContent.tsx

```diff
@@ -61,6 +66,7 @@ const BannerTextContent = forwardRef<HTMLDivElement, BannerTextContentProps>(
           variant={variant}
           inverse={colorVariant === 'main'}
           aria-label={linkText}
+          target={linkTargetBlank ? '_blank' : undefined}
```

Comment:
undefined assumes the `_self` (default) value, right?

Reply:
> undefined assumes the `_self` (default) value, right?

If the `target` attribute is set to undefined then it won't be rendered on the link element, but maybe it would be better to default to `target=_self`? I didn't realize there were so many options until I saw your other comment

Reply:
the other options are so uncommon 🤔, I'm ok with just using `_blank` option for now.